index.bs

```diff
@@ -4664,10 +4670,14 @@ The user agent must periodically run [=queue reports for delivery=] on the
 [=event-level report cache=] and [=aggregatable attribution report cache=].
 
 To <dfn>queue reports for delivery</dfn> given a [=set=] of
-[=attribution reports=] |cache|, run the following steps:
+[=attribution reports=] |cache| and a [=moment=] |now|, run the following steps:
```

Comment:
I don't think we need the `now` parameter; see the use of `[=current wall time=]` in one of the steps below.

In either case, we should be consistent.

Reply:
Good point, done.